WordPress/wordcamp.org
# Block Live Schedule: Use duration to find currently running sessions

In #299, a new field was added for session duration. We can use this to accurately get the running sessions. If a session does not have a duration, we default to `WordCamp_Post_Types_Plugin::SESSION_DEFAULT_DURATION`, 50 minutes.

Currently, the Live Schedule block does not correctly display the last session of the day, because it uses the following session to gauge what is "now." Since we have duration now, the logic for sorting and selecting "current sessions" has been updated to fetch the session that starts before now, and ends after now.

The tests have also been updated to be a bit more accurate, and the test mock data has been trimmed down to one day of sessions.

### Screenshots

In this screenshot, the "afterparty" session is happening "now", with nothing after, so we hide the "Up Next" header.

<img width="677" alt="Screen Shot 2020-01-21 at 11 11 44 AM" src="https://user-images.githubusercontent.com/541093/72821671-d48ef900-3c3e-11ea-82c9-7bc6b2d4e9bc.png">

### How to test the changes in this Pull Request:

1. Check that the tests pass
2. Set up some sessions to be running "now", or add `'nowOverride' => strtotime( 'your time' ) * 1000,` to the live schedule controller (in `add_script_data`).
3. View the live schedule and check that it is showing the right sessions
